src/model/immutable/DefaultDraftBlock.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule DefaultDraftBlock
+ * @flow
+ */
+
+'use strict';
+
+const React = require('React');
+
+const cx = require('cx');
+
+const UL_WRAP = <ul className={cx('public/DraftStyleDefault/ul')} />;
+const OL_WRAP = <ol className={cx('public/DraftStyleDefault/ol')} />;
+const PRE_WRAP = <pre className={cx('public/DraftStyleDefault/pre')} />;
+
+module.exports = {
+  'header-one': {
+    wrapper: null,
+    element: 'h1'
+  },
```

Comment:
These objects could also have a Flow type definition, to enforce their usage elsewhere in code.

```javascript
export type DraftBlockRenderConfig = {
  wrapper: ?Object;
  element: string;
};
```